honeycombio/honeycomb-opentelemetry-web
# feat(honeycomb-opentelemetry-web): Add data attributtes for LCP.

## Which problem is this PR solving?
Adds `data-` attributes to LCP web vitals and a configuration to specify which data attributes to collect. 

Closes #95 

## Short description of the changes

## How to verify that this has the expected result

Add a `data-` to the LCP element and see it's value as an attribute span under `element.data-ATTRIBUTE_NAME`

![CleanShot 2024-09-26 at 16 38 09@2x](https://github.com/user-attachments/assets/43898db1-c113-469e-816e-96cc1b215a27)
